wgpu/backends/wgpu_native/_helpers.py

```diff
@@ -332,33 +332,35 @@ def generate_report():
     report = {}
 
     report["surfaces"] = {
-        "occupied": struct.surfaces.numOccupied,
-        "vacant": struct.surfaces.numVacant,
+        "allocated": struct.surfaces.numAllocated,
+        "kept": struct.surfaces.numKeptFromUser,
+        "released": struct.surfaces.numReleasedFromUser,
         "error": struct.surfaces.numError,
         "element_size": struct.surfaces.elementSize,
     }
 
-    for backend in ("vulkan", "metal", "dx12", "dx11", "gl"):
+    for backend in ("vulkan", "metal", "dx12", "gl"):
         c_hub_report = getattr(struct, backend)
         report[backend] = {}
         for key in dir(c_hub_report):
-            c_storage_report = getattr(c_hub_report, key)
-            storage_report = {
-                "occupied": c_storage_report.numOccupied,
-                "vacant": c_storage_report.numVacant,
-                "error": c_storage_report.numError,
-                "element_size": c_storage_report.elementSize,
+            c_registry_report = getattr(c_hub_report, key)
+            registry_report = {
+                "allocated": c_registry_report.numAllocated,
+                "kept": c_registry_report.numKeptFromUser,
+                "released": c_registry_report.numReleasedFromUser,
+                "error": c_registry_report.numError,
+                "element_size": c_registry_report.elementSize,
             }
-            # if any(x!=0 for x in storage_report.values()):
-            report[backend][key] = storage_report
+            # if any(x!=0 for x in registry_report.values()):
+            report[backend][key] = registry_report
 
     return report
 
 
 class WgpuNativeCountsDiagnostics(Diagnostics):
     def get_subscript(self):
         text = ""
-        text += "    * The o, v, e are occupied, vacant and error, respecitively.\n"
+        text += "    * The a, k, r, e are allocated, kept, released, and error, respecitively.\n"
```

Comment:
Let's also fix this typo :)
```suggestion
        text += "    * The a, k, r, e are allocated, kept, released, and error, respectively.\n"
```